trymito.io/pages/chatgpt-vs-mito.tsx

```diff
@@ -0,0 +1,312 @@
+    }));
+
+    useEffect(() => {
+        const observer = new IntersectionObserver(
+            (entries) => {
+                entries.forEach((entry) => {
+                    const index = sectionRefs.current.findIndex(ref => ref.current === entry.target);
+                    if (index !== -1) {
+                        const mitoVideo = videoRefs.current[index * 2];
+                        const chatVideo = videoRefs.current[index * 2 + 1];
+
+                        if (entry.isIntersecting) {
+                            if (mitoVideo?.current && !completedVideos[index * 2]) {
+                                mitoVideo.current.play();
+                            }
+                            if (chatVideo?.current && !completedVideos[index * 2 + 1]) {
+                                chatVideo.current.play();
+                            }
+                        } else {
+                            if (mitoVideo?.current) mitoVideo.current.pause();
+                            if (chatVideo?.current) chatVideo.current.pause();
+                        }
+                    }
+                });
+            },
+            { threshold: 0.5 }
+        );
+
+        sectionRefs.current.forEach((ref) => {
+            if (ref.current) {
+                observer.observe(ref.current);
+            }
+        });
+
+        return () => {
+            sectionRefs.current.forEach((ref) => {
+                if (ref.current) {
+                    observer.unobserve(ref.current);
+                }
+            });
+        };
+    }, [completedVideos]);
+
+    return (
+        <>
+            <Head>
+                <title>ChatGPT vs Mito</title>
+                <meta name="viewport" content="width=device-width, initial-scale=1" />
+                <meta name="description" content="No more copy-pasting code that doesn&apos;t quite work. Mito lives inside Jupyter, helping you generate code based on your notebook and data." />
+            </Head>
+
+            <Header />
+
+            <div className={pageStyles.container}>
+                <main className={pageStyles.main}>
+                    <section className={titleStyles.title_card + ' ' + titleStyles.grid_card}>
+                        <h1 className={titleStyles.title}>
+                            Code faster, with fewer mistakes
+                        </h1>
+                        <p className={titleStyles.description}>
+                            No more copy-pasting code that doesn&apos;t quite work. Mito lives inside Jupyter, helping you generate code based on your notebook, your data, and your goals.
+                        </p>
+                    </section>
+
+                    {sectionsWithRefs.map((section, index) => (
+                        <section key={index} style={{ marginBottom: '0px' }}>
+                            <div className={titleStyles.description + ' ' + pageStyles.subsection_column + ' center'} style={{ marginBottom: '50px' }}>
```

Comment:
btw, we have a helpful `classNames` utility function that handles all of this class concatenation for you.